tests/php/DatabaseCleanupTest.php

```
@@ -28,10 +28,56 @@ public function test_table_map_is_correct(): void {
+					),
+				);
+			}
+		};
+
+		$result = Database_Cleanup::get_autoloaded_options( 5 );
+
+		$this->assertSame( 'small_option', $result[0]['option_name'] );
+		$this->assertSame( 10, $result[0]['size'] );
+		$this->assertSame( 'big_option', $result[1]['option_name'] );
+		$this->assertSame( 5000, $result[1]['size'] );
+	}
```

PHPMD (2.15.0)

[error] test_get_autoloaded_options_returns_sorted_rows accesses the super-global variable $GLOBALS. (undefined)

(Superglobals)